### PR TITLE
Internationalization of text messages

### PR DESCRIPTION
**Pull request by LaChope**
Implemented internationalization of text messages #91

### CODE REVIEW

**Review by blcham — CHANGES_REQUESTED** [review]
Not very managable solution. Translation variants need to be separated in a file. See how this is done e.g. in termit-ui project or [record-manager](https://github.com/opendata-mvcr/ofn-record-manager)

File: src/components/comment/CommentForm.jsx [hunk 1]
```diff
@@ -38,43 +33,46 @@ const CommentForm = (props) => {
...
+        <Col className="col-lg-12 p-0">
+          <Row className="container-fluid p-0 m-0">
+            <div id="comment-form">
+              <Form.Control
+                className="comment-form-control"
+                name="comment"
+                as="textarea"
+                placeholder={intl.formatMessage({
+                  id: 'comment.form.placeholder',
+                  defaultMessage: 'Write your comments here  (Ctrl+Enter to confirm, Esc to cancel)'
```
Comment (blcham) [inline src/components/comment/CommentForm.jsx:58]: @LaChope 

What is the purpose of the defaultMessage? Isn't it useful to not write the same string twice? Whouldn't we say that "en" variant is the default here without need to specify "defaultMessage" ?

**Review by blcham — APPROVED** [review]
LGTM, although there is one comment that would be nice to resolve

File: src/components/comment/CommentForm.jsx (hunk 1 above)
Comment (LaChope) [reply src/components/comment/CommentForm.jsx:58]: @blcham It is just to cover the case where the option "locale" would be an empty string. Is it useful scenario to take into account or should we dismiss it? If we dismiss it defaultMessage is not necessary

Comment (blcham) [reply src/components/comment/CommentForm.jsx:58]: Thus I would assume we always have at least one "locale" complete -- we can consider 'en' locale for that case.

**Pull request merged**